components/Registration/index.tsx

```diff
@@ -290,26 +333,55 @@ const Registration: FunctionComponent<RegistrationProps> = ({
   };
 
   const showRegistrationSuccess = useMemo(
-    () => userRegistered && activeWindow?.airdrop_window_status === WindowStatus.REGISTRATION,
+    () =>
+      userRegistered &&
+      activeWindow?.airdrop_window_status === WindowStatus.REGISTRATION,
     [userRegistered, activeWindow]
   );
 
+  if (!activeWindow) {
+    return null;
+  }
+
+  if (!account) {
+    return (
+      <Grid container spacing={2} px={4} mt={2} mb={8}>
+        <Grid item xs={12} sm={6}>
+          <Airdropinfo blogLink="www.google.com" />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <AirdropRegistrationMini
+            startDate={
+              new Date(
+                `${activeWindow.airdrop_window_registration_start_period} UTC`
```

Comment:
The date displayed in the AirdropRegistrationMini should be based on the current time and window. 
It should not be always registration_start.

Reply:
Fixed in the latest commit